include/ed/update_request.h

```diff
@@ -51,6 +51,30 @@ class UpdateRequest
     void setShape(const UUID& id, const geo::ShapeConstPtr& shape) { shapes[id] = shape; flagUpdated(id); }
 
 
+    //VOLUMES
+
+    std::map<UUID, std::map<std::string, geo::ShapeConstPtr> > volumes_added;
```

Comment:
No `_` at the end?

Also, would be nice if you explain the idea, as I don't understand why you track added volumes and removed volumes.

Reply:
Because these need to be added/removed. Just like type. No underscore used in update_request at all. So out of this scope.

Reply:
I don't understand your explanation, but fair enough for now.